lib/geometry/arc.rb

```diff
@@ -34,13 +39,13 @@ class Arc
 	# @option options [Numeric] :end	Ending angle
 	# @return [ThreePointArc]
 	def self.new(options={})
```

Comment:
Hmm, this is interesting. Having `Arc.new` return a `ThreePointArc` kinda steps on principle of least surprise :thinking: